src/gui/Src/Utils/StringUtil.cpp

```diff
@@ -224,4 +226,23 @@ QString ToDateString(const QDate & date)
         "Dec"
     };
     return QString().sprintf("%s %d %d", months[date.month() - 1], date.day(), date.year());
+    */
+    return QLocale(QString(currentLocale)).toString(date);

```

Comment:
I actually did this for the reason of bug reporting. I want my date format in every screenshot so I can see what version people are using.